### ISSUE TITLE
Stop using Google CDNs due to GFW

### ISSUE THREAD

**Issue by TheBlueMatt** [bitcoin-core/bitcoincore.org#396]
GFW can sometimes block Google CDNs (ajax.googleapis which we use for jquery is a common issue), so we shouldn't be using that.

**Comment by harding** [comment]
We currently load two files from `*.googleapis.com`: JQuery and some fonts.

For the JQuery, we fall back to loading from ourselves if not loaded from the remote server ([source](https://github.com/bitcoin-core/bitcoincore.org/blob/gh-pages/_includes/_scripts.html#L7)):

```html
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="{{ site.url }}/assets/js/vendor/jquery-1.9.1.min.js"><\/script>')</script>
```

I verified this works in production on Firefox at least by blocking the Google server and then loading the live site:

![2017-07-05-12_31_14_802153895](https://user-images.githubusercontent.com/61096/27874674-57314db6-617e-11e7-8a4e-434b31b088a1.png)

For the fonts, I also blocked those and looked at several pages (including pages with both simplified and traditional Chinese text), and the only change I see is that the fonts are a bit uglier and a bit wider, so I don't think we need any change there for GFW blocking issues.  For example, first screenshot below is the regular site and the second screenshot is with the font host blocked:

![2017-07-05-12_35_42_478302892](https://user-images.githubusercontent.com/61096/27875198-5e4d3f90-6180-11e7-94fc-c1ea2e4efff2.png)

![2017-07-05-12_51_33_161193284](https://user-images.githubusercontent.com/61096/27875316-b9a267f8-6180-11e7-9221-a7dfb1d4434e.png)

**Comment by TheBlueMatt** [comment]
There's the additional issue that loading from Google slows you down if you don't have it cached (because initializing a whole new TLS connection takes quite a few RTTs). The site loads really quite slowly for me if I disable cache, and a big chunk is waiting on jQuery to load. I really don't think it's worth the hassle.

**Comment by harding** [comment]
@TheBlueMatt that seems reasonable.  I opened https://github.com/bitcoin-core/bitcoincore.org/pull/401 for the JQuery issue.  Do you also want fonts to be loaded from the BitcoinCore.org server?

**Comment by btcdrak** [comment]
No problem with locally stored (although it usually does affect user experience) but those particular CDN urls  are not affected by GFW.

**Comment by btcdrak** [comment]
Closed in #401
